client/post-editor/editor-drawer/index.jsx

```diff
@@ -296,26 +304,26 @@ const EditorDrawer = React.createClass( {
 
 		return (
 			<div className="editor-drawer">
-				{ site && ! this.currentPostTypeSupportsAll() && (
+				{ site && ! POST_TYPE_SUPPORTS[ type ] && (
```

Comment:
I think it's a bit clearer if you called a function like `! supportsAllFeatures( type )` rather than depending on the structure of fallback object.